ansible/roles/microshift-start/tasks/main.yml

```diff
@@ -56,11 +56,11 @@
   delay: 1
 
 - name: wait for pods running
-  ansible.builtin.shell: kubectl get po -A --no-headers | grep Running | wc -l
+  ansible.builtin.shell: oc get po -A --no-headers | grep Running | wc -l
   retries: 60
   delay: 1
   register: result
-  until: result.stdout == "6"
+  until: result.stdout == "8"
```

Comment:
This task seems to be duplicated in at least 2 places. Can we refactor into a common file and call it?

Reply:
Agreed, pulled into common task.